src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -677,11 +677,11 @@ protected override void OnNodePropertyChanged(object sender, PropertyChangedEven
 
         public override void GenerateViewGeometryFromRenderPackagesAndRequestUpdate(IEnumerable<IRenderPackage> taskPackages)
         {
-            foreach (var p in taskPackages)
+            /*foreach (var p in taskPackages)
             {
                 Debug.WriteLine(string.Format("Processing render packages for {0}", p.Description));
             }
-
+            */
```

Comment:
If code not meant for release, we usually just remove them from any submission. If developer need it for debugging, they should add themselves locally. If you really want to keep this, format the info in a nicer way and put in the function description like:
        /// <summary>
        /// xxxxxx
        /// </summary>
        /// <param name="taskPackages"></param>